src/docs/introduction.rst

```diff
@@ -1,80 +1,97 @@
 remove a user. However, that’s pretty mean and a waste of time when there
 is a much easier way of doing things.
 
-The community project FreeIPA *(Free Identity Policy and Auditing)* is
-an Identity Management solution. It’s also known as Red Hat IdM, which
-is the downstream name. FreeIPA, Red Hat IdM uses 389DS
-*(Directory Server)* for its database.
+Identity management solutions
+*****************************
+
+The community project `FreeIPA`_ is an Identity Management solution. It’s also
+known as the Red Hat IdM or simply IPA. FreeIPA uses 389 Directory Server as its
+database. Directory servers contain objects like a user object. That user object
+contains user information stored in LDAP attributes and it is accessible on the
+network through the `LDAP`_ *(Lightweight Directory Access Protocol)* protocol.
+
+`Microsoft Windows Server`_ implements a directory service named `Microsoft
+Active Directory Domain Services`_, abbreviated as Active Directory or AD.
+Active Directory is another identity management solution that is specifically
+designed for Windows. It also uses an LDAP server and has many similarities with
+IPA.
+
+Beside FreeIPA and Active Directory, SSSD can also integrate to other identity
+solutions using the LDAP provider (for pure LDAP servers) and the Kerberos
+provider (for Kerberos authentication instead of plain passwords). It can also
+directly integrate with local users using the files provider.
+
+.. note::
+
+    Do not confuse LDAP with 389 Directory Server. 389 Directory Server is one
+    of the projects that implements the LDAP protocol.
 
-Directory servers contain objects like a user object, that user object
-contains user information stored in LDAP attributes and it is accessible
-on the network the LDAP *(Lightweight Directory Access Protocol)*
-network protocol.
+.. seealso::
 
-*Please do not confuse LDAP with 389DS, 389DS is a directory server that
-uses the LDAP protocol on port 389.*
+    Checkout the rest of this website and following manual pages for more
+    information on the different kinds of providers.
 
-Microsoft Windows Server implements a directory service named Microsoft
-*ADDS* (Active Directory Domain Services), abbreviated as Microsoft AD.
-AD is another identity management solution for Windows. It has
-an LDAP server and shares many components with IPA. IPA and AD can have
-a trust created between the domain domains. Why? Consider if there was a
-company merger or acquisition. There won’t be a need to migrate users but
-instead, set up a trust and allow Bob from industry giant "SSSD Corp" to
-administer newly acquired small business "LDAP LLC".
+    - `sssd-ldap <https://www.mankier.com/5/sssd-ldap>`_
+    - `sssd-krb5 <https://www.mankier.com/5/sssd-krb5>`_
+    - `sssd-ipa <https://www.mankier.com/5/sssd-ipa>`_
+    - `sssd-ad <https://www.mankier.com/5/sssd-ad>`_
+    - `sssd-files <https://www.mankier.com/5/sssd-files>`_
+
```

Comment:
It might be better to tell the user to reference man pages on their own systems, instead of linking to mankier.com. This will ensure all options are relevant to the SSSD version they are using.

Reply:
I changed it to 'man sssd-ipa' to refer to the man command. Otherwise can you suggest an uninterruptive wording?